src/Microsoft.Framework.PackageManager/Program.cs

```diff
@@ -61,7 +61,9 @@ public int Main(string[] args)
             {
                 c.Description = "Restore packages";
 
-                var argRoot = c.Argument("[root]", "Root of all projects to restore. It can be a directory, a project.json, or a global.json.");
+                var argRoot = c.Argument("[root]",
+                    "Root of projects to restore. A value can be a path to directory/project.json/global.json. Accepts multiple values.",
+                    multipleValues: true);
```

Comment:
I am going to make the change mentioned by @davidfowl below. So if there is no `project.json` in given folder, we recursively traverse all subfolders and restore for all `project.json` we see. So we need to revise the help info a little bit:
```
List of projects and project folders to restore. The values can be: a path to project.json or global.json file, or a folder having project.json in subfolders
```